schemas/manifest-schema.json

```diff
@@ -0,0 +1,376 @@
+          "minItems": 1
+        }
+      },
+      "required": ["files", "importPaths"],
+      "unevaluatedProperties": false
+    },
+    "binaries" : {
+      "title": "binaries",
+      "description": "A binary\nhttps://substreams.streamingfast.io/reference-and-specs/manifests#binaries",
+      "type": "object",
+      "properties": {
+        "default" : {
+          "title": "default binary",
+          "description": "the default binary",
+          "type": "object",
+          "properties": {
+            "type": {
+              "title": "binary type",
+              "description": "A binary type\nhttps://substreams.streamingfast.io/reference-and-specs/manifests#binaries-name-.type",
+              "enum": ["wasm/rust-v1"]
+            },
+            "file": {
+              "title": "binary file",
+              "description": "A binary file\nhttps://substreams.streamingfast.io/reference-and-specs/manifests#binaries-name-.file",
+              "type": "string"
+            },
+            "protoPackageMapping" : {
+              "title": "binary protoPackageMapping",
+              "description": "a protoPackageMapping",
+              "type": "object",
+              "propertyNames": {
+                "pattern" : {
+                  "type": "string"
+                }
+              },
+              "unevaluatedProperties": false,
+              "minProperties": 1
+            }
+          },
+          "required": ["type", "file"],
+          "additionalProperties": false
+        }
+      },
+      "required": ["default"],
+      "unevaluatedProperties": false
+    },
+    "modules": {
+      "title": "modules",
+      "description": "A module \nhttps://substreams.streamingfast.io/reference-and-specs/manifests#modules",
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "name": {
+            "description": "A module name\nhttps://substreams.streamingfast.io/reference-and-specs/manifests#module-name",
+            "type": "string"
+          },
+          "initialBlock": {
+            "description": "A module initialBlock\nhttps://substreams.streamingfast.io/reference-and-specs/manifests#module-initialBlock",
+            "type": "number"
+          },
+          "kind": {
+            "description": "A module kind\nhttps://substreams.streamingfast.io/reference-and-specs/manifests#module-kind",
+            "enum": [
+              "map",
+              "store"
+            ]
+          },
+          "binary": {
+            "description": "A module binary\nhttps://substreams.streamingfast.io/reference-and-specs/manifests#module-binary",
+            "type": "string"
+          },
+          "inputs": {
+            "description": "A module input\nhttps://substreams.streamingfast.io/reference-and-specs/manifests#module-inputs",
+            "type": "array",
+            "items": {
+              "type": "object",
+              "properties": {
+                "store": {
```

Comment:
Isn't it one of `map` or `store`?

Reply:
no: https://substreams.streamingfast.io/reference-and-specs/manifests#module-inputs

Reply:
It's definitely one of thing it's either `{ "source" }` or `{ "store", "mode" }` or `{ "map" }`, let's discuss this tomorrow.